documentation/api-scenario/how-to/IntegrateWithTestProxy.md

```diff
@@ -0,0 +1,87 @@
+# How to integrate test-proxy with API Scenario test
+
+In this section, we'll show you how to make test-proxy recording with API Scenario test, assuming you've already learnt how to use API Scenario test. If not, please go to [QuickStart](./QuickStart.md).
```

Comment:
why make recording? the article does not make clarity of why.

Reply:
added some explanation in the second paragraph. This documentation is mainly for how to make recording.